libs/sdk-backend-bear/src/backend/workspace/catalog/factory.ts

```diff
@@ -270,24 +279,20 @@ export class BearWorkspaceCatalogFactory implements IWorkspaceCatalogFactory {
 
         const bearCatalogAttributes = bearCatalogItems.filter(GdcCatalog.isCatalogAttribute);
         const attributeUris = bearCatalogAttributes.map((attr) => attr.links.self);
-        const displayFormUris = flatMap(bearCatalogAttributes, (attr) => {
-            const geoPins = attr.links.geoPinDisplayForms ?? [];
```

Comment:
So the geo display forms are now handled in the `extractDisplayFormsFromBearAttributes` right?

Reply:
Yes